cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskPlannerTest.java

```diff
@@ -112,14 +116,21 @@ public void testGetPartitionMovementTasks() {
     proposals.add(partitionMovement2);
     proposals.add(partitionMovement3);
     proposals.add(partitionMovement4);
-    ExecutionTaskPlanner planner = new ExecutionTaskPlanner();
+    // Test different execution strategies.
+    ExecutionTaskPlanner basePlanner = new ExecutionTaskPlanner(Collections.singletonList(BaseReplicaMovementStrategy.class.getName()));
```

Comment:
If we could ensure that `BaseReplicaMovementStrategy` is not a config, we can skip passing it as a parameter in all cases (please see prev. discussion).